CHANGELOG.rst

```diff
@@ -3,6 +3,9 @@ VERSIONS
 
 Next Release
 ------------
+
```

Comment:
Looks like I forgot to push after merging my release branch to develop. Shouldn't cause any issues.